styleguide/screens/Patterns/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,156 @@
+        <T elm="p" className={ cx(m.mtr, m.largeI, m.demi) }>
+          A <C>tooltip</C> provides additional information to an item on user
+          interaction (click or hover).  They’re typically used add more context to the element
+          in question.  They should only be used when unable to surface enough information to the
+          user.  Try to avoid providing critical information to the user like this, as the
+          additional interaction as this isn’t great for usability on touch devices.
+        </T>
+        <D>
+          <H level={ 2 } className={ m.mtr }>Default Tooltip</H>
+          <T elm="p" className={ m.mtr }>
+            Our default <C>tooltip</C> comes in black. Attach it to a specific element, by proving
+            that element in the target prop.
+          </T>
+          <Specimen
+            classNames={ {
+              root: m.mtr,
+              specimenContainer: cx(m.par, css.specimenContainer),
+            } }
+            code={ dedent`
+              <Tooltip
+                target={ <DefaultTarget /> }
+                verticalAttachment={ VERTICAL_ATTACHMENTS.TOP }
+                horizontalAttachment={ HORIZONTAL_ATTACHMENTS.CENTER }
+                active
+              >
+                <Content />
+              </Tooltip>
+            ` }
+          >
+            <Tooltip
+              target={
+                <DefaultTarget
+                  onMouseEnter={ this.handleMouseEnter }
+                  onMouseLeave={ this.handleMouseLeave }
+                />
+              }
+              verticalAttachment={ VERTICAL_ATTACHMENTS.TOP }
+              horizontalAttachment={ HORIZONTAL_ATTACHMENTS.CENTER }
+              active={ showDefaultTooltip }
+            >
+              <Content />
+            </Tooltip>
+          </Specimen>
+        </D>
+        <D>
+          <H level={ 2 } className={ m.mtr }>Light Tooltip</H>
+          <T elm="p" className={ m.mtr }>
+            If you’re using a tooltip on a dark background, use the light variant.
+          </T>
+          <Specimen
+            classNames={ {
+              root: m.mtr,
+              specimenContainer: cx(m.par, css.specimenContainer),
+            } }
+            code={ dedent`
+              <Tooltip
+                target={ <DefaultTarget /> }
+                verticalAttachment={ VERTICAL_ATTACHMENTS.TOP }
+                horizontalAttachment={ HORIZONTAL_ATTACHMENTS.CENTER }
+                active
+              >
+                <Content />
+              </Tooltip>
+            ` }
+            variant="dark"
+          >
+            <Tooltip
+              target={
+                <LightTarget
+                  onMouseEnter={ this.handleMouseEnter }
+                  onMouseLeave={ this.handleMouseLeave }
+                />
+              }
+              verticalAttachment={ VERTICAL_ATTACHMENTS.CENTER }
+              horizontalAttachment={ HORIZONTAL_ATTACHMENTS.RIGHT }
```

Comment:
We don't need to show different attachments as part of this documentation—the link top the Tether documentation will suffice